app/v1/controllers/requestParsers/validators/AmendSelfEmploymentAnnualSummaryValidator.scala

```diff
@@ -0,0 +1,151 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.controllers.requestParsers.validators
+
+import v1.controllers.requestParsers.validators.validations._
+import v1.models.errors._
+import v1.models.request.amendAnnualSummary._
+
+class AmendSelfEmploymentAnnualSummaryValidator extends Validator[AmendAnnualSummaryRawData] {
+
+  private val validationSet = List(parameterFormatValidation, parameterRuleValidation, bodyFormatValidation, bodyFieldValidation)
+
+  private def parameterFormatValidation: AmendAnnualSummaryRawData => List[List[MtdError]] = (data: AmendAnnualSummaryRawData) => {
+    List(
+      NinoValidation.validate(data.nino),
+      BusinessIdValidation.validate(data.businessId),
+      TaxYearValidation.validate(data.taxYear)
+    )
+  }
+
+  private def parameterRuleValidation: AmendAnnualSummaryRawData => List[List[MtdError]] = { data =>
+    List(
+      TaxYearNotSupportedValidation.validate(data.taxYear)
+    )
+  }
+
+  private def bodyFormatValidation: AmendAnnualSummaryRawData => List[List[MtdError]] = { data =>
+    List(
+      JsonFormatValidation.validate[AmendAnnualSummaryBody](data.body)
+    )
+  }
+
+  private def bodyFieldValidation: AmendAnnualSummaryRawData => List[List[MtdError]] = { data =>
+    val body = data.body.as[AmendAnnualSummaryBody]
+    val errorsO: List[List[MtdError]] = List(
+      body.adjustments.map(validateAdjustments).getOrElse(Nil),
+      body.allowances.map(validateAllowances).getOrElse(Nil),
+      body.nonFinancials.map(validateNonFinancials).getOrElse(Nil)
+    ).flatten
+    List(Validator.flattenErrors(errorsO))
+  }
+
+  private def validateAdjustments(adjustments: Adjustments): List[List[MtdError]] = {
+    List(
+      NumberValidation.validateOptional(
+        field = adjustments.includedNonTaxableProfits,
+        path = s"/adjustments/includedNonTaxableProfits"
+      ),
+      NumberValidation.validateOptional(
+        field = adjustments.basisAdjustment,
+        path = s"/adjustments/basisAdjustment"
+      ),
+      NumberValidation.validateOptional(
+        field = adjustments.overlapReliefUsed,
+        path = s"/adjustments/overlapReliefUsed"
+      ),
+      NumberValidation.validateOptional(
+        field = adjustments.accountingAdjustment,
+        path = s"/adjustments/accountingAdjustment"
+      ),
+      NumberValidation.validateOptional(
+        field = adjustments.averagingAdjustment,
+        path = s"/adjustments/averagingAdjustment"
+      ),
```

Comment:
Number Validation seems to only account for values between 0 and 99999999999.99 while this field allows values between -99999999999.99 and 99999999999.99